test/cpp/opus_codec_test.cpp

```
@@ -0,0 +1,206 @@
+    int sz1 = enc.encode(pcmIn.data(), frameSize, pkt1,
+                         audio_config::kMaxOpusPacketSize);
+    assert(sz1 > 0);
+    int sz2 = enc.encode(pcmIn.data(), frameSize, pkt2,
+                         audio_config::kMaxOpusPacketSize);
+    assert(sz2 > 0);
+
+    // Simulate a lost pkt1: skip decode(pkt1) and call decodeFec(pkt2)
+    // directly. Opus uses either its FEC side-channel or PLC internally.
+    OpusDecoder dec;
+    std::vector<int16_t> fecOut(frameSize, 0);
+    int fecSamples = dec.decodeFec(pkt2, sz2, fecOut.data(), frameSize);
```

Clang (14.0.6)

[warning] variable 'fecSamples' is not initialized

(cppcoreguidelines-init-variables)